src/controllers/auth.controller.js

```diff
@@ -0,0 +1,107 @@
+import { ApiError } from '../exceptions/api.error.js';
+import { User } from '../modules/User.js';
+import { userService } from '../service/user.service.js';
+import bcrypt from 'bcrypt';
+import { Console } from '../utils/Console.js';
+import { jwtService } from '../service/jwt.service.js';
+import { tokenService } from '../service/token.service.js';
+
+const register = async (req, res) => {
+  const { userName, email, password } = req.body;
+
+  const errors = {
+    email: userService.validateEmail(email),
+    password: userService.validatePassword(password),
+  };
+
+  if (errors.email || errors.password) {
+    throw ApiError.badRequest('Email, password, are not validate', errors);
```

Comment:
The error message in `ApiError.badRequest` is not grammatically correct. Consider changing it to 'Email and password are not valid.'